src/Option_Alpha/web/routes/debate.py

```
@@ -0,0 +1,186 @@
+@router.post("/{symbol}", status_code=202, response_model=DebateStarted)
+async def start_debate(
+    symbol: Annotated[str, Depends(validate_ticker_symbol)],
+    repo: Annotated[Repository, Depends(get_repository)],
+    market_service: Annotated[MarketDataService, Depends(get_market_data_service)],
+) -> DebateStarted:
+    """Start an AI debate for the given ticker symbol.
+
+    Kicks off the debate pipeline as a background task and returns immediately.
+    Poll GET /api/debate or use the ticker's debate history to retrieve results.
+    """
+    asyncio.create_task(_run_debate_task(symbol, market_service, repo))
```

Ruff (0.15.1)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)